src/api/app/views/webui2/webui/project/_delete_dialog.html.haml

```diff
@@ -0,0 +1,15 @@
+.modal.fade#delete-modal{ tabindex: -1, role: 'dialog', aria: { labelledby: 'delete-modal-label', hidden: true } }
```

Comment:
I would call the id more specific like ``project-delete-modal-id`` otherwise it could collide in the future.